schema/vrs.yaml

```diff
@@ -546,8 +546,7 @@ definitions:
     deprecated: true
     description: >-
       DEPRECATED: A SimpleInterval represents a span of sequence. Positions are
-      always represented by contiguous spans using interbase
-      coordinates.
+      always represented by contiguous spans using interbase coordinates.
 
       This class is deprecated. Use SequenceInterval instead.
```

Comment:
It is a little sad that we'll be turning around and DEPRECATING SequenceInterval in v2, but I guess we need to draw the line somewhere. sigh.